testgrid/config/config.yaml

```diff
@@ -2716,6 +2724,10 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gci-gke
   - name: aws
     test_group_name: ci-kubernetes-e2e-kops-aws
+  - name: gce-device-plugin-gpu
```

Comment:
Maybe let them run and be in passing state for a week before adding them to release-master-blocking dashboard?